src/lake/Lake/Build/Fetch.lean

```diff
@@ -4,8 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Mac Malone
 -/
 prelude
-import Lake.Util.Lift
-import Lake.Util.Error
+import Lake.Util.Log
```

Comment:
Out of curiosity, why change the imports here? Also, why remove `Lift`?

Reply:
Experimentation showed that this allowed me to `import Lake.Build.Fetch`, which allowed there to be documentation for `FetchM`, and all the tests passed. My technique was "minimize the import graph until the manual builds and the tests pass", rather than "understand all the different imports and why they're here".

Reply:
Fair enough. The odd part is that `Lake.Util.Log` is a strict superset of the two previous imports (it imports both), so this does not actually shrink the import graph.

Reply:
Weird. I just tested with these two lines reverted, and it worked. I'll undo this change. Thanks!